index.js

```diff
@@ -320,7 +322,8 @@ async function format (opts, pkg = {}) {
 
 module.exports.write = write
 // TODO: look at https://npm.im/json-file-plus for writing
```

Comment:
We should look at these as well: 

- https://www.npmjs.com/package/detect-indent
- https://www.npmjs.com/package/detect-newline


They are used in `standard-version` and that has appeared to work well for me